learning/tour-of-beam/learning-content/io/big-query-io/read-table/python-example/task.py

```diff
@@ -56,7 +52,7 @@ def run(argv=None):
 
 
     with beam.Pipeline(options=pipeline_options, argv=argv) as p:
-        (p | 'ReadFromBigQuery' >> beam.io.ReadFromBigQuery(query='select * from `apache-beam-testing.clouddataflow_samples.weather_stations`',
+      (p | 'ReadFromBigQuery' >> beam.io.ReadFromBigQuery(table='apache-beam-testing:clouddataflow_samples.weather_stations',
```

Comment:
I think indentation got changed to 2 spaces here

Reply:
Spaces are necessary for the python file to work. @olehborysevych knows about the GCP error

Reply:
Remove spaces are necessary for the python file to work. @olehborysevych   knows about the GCP error

Reply:
@olehborysevych is there any issue created for the error?

Reply:
@sirenbyte can you comment on the known GCP error? we can merge this once we know the cause of failure

Reply:
great thanks for the info @olehborysevych